core/services/ocr2/delegate.go

```diff
@@ -403,6 +403,17 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) ([]job.ServiceCtx, error) {
 			return nil, errors.Wrap(err2, "could not build dependencies for ocr2 keepers")
 		}
 
+		var cfg ocr2keeper.PluginConfig
```

Comment:
I don't think these should be a part of job spec. You can't change a job spec without a DB migration, or you need to delete and recreate a job. We should move these to env vars